simulator-spring-boot/src/test/java/org/citrusframework/simulator/model/AbstractAuditingEntityIT.java

```diff
@@ -0,0 +1,41 @@
+package org.citrusframework.simulator.model;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import lombok.Setter;
+import org.assertj.core.data.Offset;
+import org.assertj.core.data.TemporalUnitLessThanOffset;
+import org.citrusframework.simulator.common.TimeProvider;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import java.time.ZoneId;
+import java.time.ZonedDateTime;
+
+import static java.time.temporal.ChronoUnit.SECONDS;
+import static lombok.AccessLevel.NONE;
+import static org.assertj.core.api.Assertions.assertThat;
+
+class AbstractAuditingEntityIT {
+
+    public static final TemporalUnitLessThanOffset LESS_THAN_5_SECONDS = new TemporalUnitLessThanOffset(1, SECONDS);
```

Comment:
I think the name does not match what it is indicated to do.. switch either to `LESS_THAN_1_SECOND` or `...(5, SECONDS);`. right?